polygon/heimdall/service.go

```diff
@@ -59,6 +59,10 @@ func AssembleService(borConfig *borcfg.BorConfig, heimdallUrl string, dataDir st
 }
 
 func NewService(borConfig *borcfg.BorConfig, client HeimdallClient, store ServiceStore, logger log.Logger) Service {
+	return newService(borConfig, client, store, logger)
```

Comment:
What's the need for the extra function?

Reply:
I would like to access the private `*service` struct in the test
`NewService` returns a `Service` interface
`newService` returns a `*service` pointer to unexpected struct